test/fakes/fakedatastore/fakedatastore.go

```diff
@@ -347,6 +347,19 @@ func (s *DataStore) ListRegistrationEntries(ctx context.Context,
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
+	// no pagination allow for this fake, for now it return only one page
```

Comment:
should we implement pagination in the fake datastore so the server startup code that paginates entries for trust domain validation can be tested?

Reply:
added